### ISSUE TITLE
Disable Wake upon open

### ISSUE THREAD

**Issue by Horcus87** [Kemmey/Kemmey-TeslaWatch-Public#218]
When app is opened, it automatically wakes the Tesla. Would be great this can be disabled by default, and will only Wake after complication or in-app toggle

**Comment by Kemmey** [comment]
Hey @Horcus87 

The wake on launch is in line with the Tesla iOS app, and is required for the app to show a valid state. Much could have happened since the last time the app received state info from vehicle. Some actions might not even be allowed in this unknown state. So first priority on launch is to get a fresh vehicle state to update your to current conditions. This also has the added consequence that any actions that might be triggered will execute much faster as the car will be primed already. 

So basically, this is a will not fix, sorry.

**Comment by mallo3w** [comment]
Totally understand the above reason why this isn't a thing, but accidentally knocking the complication and inadvertently waking the car is somewhat a nuisance, but I can forgive as the app in general is absolutely brilliant! 
Elon not offered you a job yet!?
